### PR TITLE
small tweaks

### PR DESCRIPTION
**Pull request by dougmoscrop**

### CODE REVIEW

File: lib/Runner.js [hunk 1]
```diff
@@ -61,30 +66,34 @@ class Runner extends EventEmitter {
           })
           .promise()
           .then(response => {
+            if (timer === 'wall') {
+              const duration = performance.now() - before;
```
Comment (AntonBazhal) [inline lib/Runner.js]: Why not just `return`?

File: lib/Runner.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
 'use strict';
 
-const AWS = require('aws-sdk');
 const EventEmitter = require('events');
+const https = require('https');
```
Comment (AntonBazhal) [inline lib/Runner.js]: Why two groups?

Comment (dougmoscrop) [reply lib/Runner.js]: I usually separate 

- built ins
- third party
- locals

do you want to just combine builtin and third party?

File: lib/Runner.js (hunk 1 above)
Comment (dougmoscrop) [reply lib/Runner.js]: oops, left over from whenN I was doing it differently

**Review by AntonBazhal — APPROVED** [review]

**Pull request merged**